src/api/lambdas/bedrock-api-backend/assets/getTasks.js

```diff
@@ -1,18 +1,19 @@
 /* eslint-disable no-console */
-import pgpkg from 'pg';
 import pgErrorCodes from '../pgErrorCodes.js';
-import { getInfo } from '../utilities/utilities.js';
+import { newClient, checkExistence } from '../utilities/utilities.js';
 
-const { Client } = pgpkg;
-
-async function newClient(connection) {
-  const client = new Client(connection);
+async function getInfo(client, idField, idValue, name, tableName) {
+  // Querying database to get information. Function can be used multiple times per method
+  // if we need information from multiple tables
+  const sql = `SELECT * FROM ${tableName} where ${idField} like $1`;
```

Comment:
Unrelated to the PR, but wondering why the SQL is using "like" instead of equality for ID comparison.